hub-templates/basehub/templates/nfs-ganesha.yaml

```diff
@@ -0,0 +1,13 @@
+{{ if .Values.nfsGanesha.enabled }}
```

Comment:
```suggestion
{{- if .Values.nfsGanesha.enabled -}}
```

Reply:
My rules of thumb:
- always chomp left
- always chomp right before the first content to be rendered in a YAML file